manager/volume.go

```diff
@@ -258,7 +259,7 @@ func (m *VolumeManager) Create(name string, spec *types.VolumeSpec) (v *longhorn
 
 	v = &longhorn.Volume{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
+			Name: strings.ToLower(name),
```

Comment:
At existing VolumeCreate, Longhorn will validate if the volume name is compliant with DNS (RFC 1123). So I assume there shouldn't be any existing volume named out of the boundary.